src/EditorFeatures/Core/Classification/Syntactic/SyntacticClassificationTaggerProvider.TagComputer.cs

```diff
@@ -109,8 +111,16 @@ internal partial class TagComputer
 
             public event EventHandler<SnapshotSpanEventArgs>? TagsChanged;
 
-            private IClassificationService? TryGetClassificationService(ITextSnapshot snapshot)
-                => _workspace?.Services.SolutionServices.GetProjectServices(snapshot.ContentType)?.GetService<IClassificationService>();
+            private (SolutionServices solutionServices, IClassificationService classificationService)? TryGetClassificationService(ITextSnapshot snapshot)
+            {
+                if (_workspace?.Services.SolutionServices is not { } solutionServices)
+                    return null;
```

Comment:
This should not be possible afaict. Should always be non null.

Reply:
sorry, phone review.  ignore this.  i though it wsa checking SOlutionServices for null, not the workspace.